benchmarks/nemo/tracer_advection/original/Makefile

```diff
@@ -21,39 +27,53 @@ DL_TIMER_NAME = libdl_timer_omp.a
 # Shorthand for invoking PSyclone.
 PSYCLONE = psyclone -api nemo -l output
 
-tra_adv_serial: dl_timer
+tra_adv_serial: dl_timer ./tra_adv.F90
 	mkdir -p $@
 	cp tra_adv.F90 $@/.
 	cp Makefile_gen $@/Makefile
-	${MAKE} PROF_LIB_INC="-I../${DL_TIMER_DIR}/src" \
-            LDFLAGS="${LDFLAGS} ../${DL_TIMER_DIR}/${DL_TIMER_NAME}" -C $@
+	${MAKE} FORT_FLAGS="${F90FLAGS} -I../${DL_TIMER_DIR}/src" \
+            LDFLAGS="${LDFLAGS} ${OMPFLAGS} ../${DL_TIMER_DIR}/${DL_TIMER_NAME}" -C $@
 
 # The OpenACC-accelerated version has the existing dl_timer calls but also
 # has profiling instrumentation added that uses NVTX.  We profile 'invokes'
 # rather than 'kernels' since the whole outer loop gets put into a single
 # ACC KERNELS region.
-tra_adv_acc: dl_timer
+tra_adv_acc: dl_timer ./tra_adv.F90
 	mkdir -p $@
 	${PSYCLONE} --profile invokes -s ../scripts/kernels_trans.py -opsy $@/tra_adv.f90 ./tra_adv.F90
 	cp Makefile_gen $@/Makefile
-	${MAKE} PROF_LIB_INC="-I${PSYCLONE_NVIDIA_LIB_DIR} -I../${DL_TIMER_DIR}/src" \
+	${MAKE} FORT_FLAGS="${F90FLAGS} -I${PSYCLONE_NVIDIA_LIB_DIR} -I../${DL_TIMER_DIR}/src" \
            LDFLAGS="${LDFLAGS} ../${DL_TIMER_DIR}/${DL_TIMER_NAME} ${PSYCLONE_NVIDIA_LIB_DIR}/libnvtx_prof.a" -C $@
 
-tra_adv_omp: ../scripts/omp_trans.py
+tra_adv_omp: ../scripts/omp_trans.py dl_timer ./tra_adv.F90
 	mkdir -p $@
 	${PSYCLONE} -s ../scripts/omp_trans.py -opsy $@/tra_adv.f90 ./tra_adv.F90
 	cp Makefile_gen $@/Makefile
-	${MAKE} PROF_LIB_INC="-I../${DL_TIMER_DIR}/src" \
+	${MAKE} FORT_FLAGS="${F90FLAGS} ${OMPFLAGS} -I../${DL_TIMER_DIR}/src" \
             LDFLAGS="${LDFLAGS} ../${DL_TIMER_DIR}/${DL_TIMER_NAME}" -C $@
 
+tra_adv_omp_gpu: ../scripts/omp_gpu_trans.py dl_timer ./tra_adv.F90
+ifndef UMEMFLAGS
+	$(error The OMP offload target requires OpenMP unified memory but the UMEMFLAGS environment variable is not set)
+endif
+	mkdir -p $@
+	${PSYCLONE} -s ../scripts/omp_gpu_trans.py -opsy $@/tra_adv.f90 ./tra_adv.F90
+	cp Makefile_gen $@/Makefile
+	${MAKE} PROF_LIB_INC="-I../${DL_TIMER_DIR}/src" \
+            FORT_FLAGS="${F90FLAGS} ${OMPTARGETFLAGS} ${UMEMFLAGS} -I../${DL_TIMER_DIR}/src" \
+            LDFLAGS="${LDFLAGS} ${OMPTARGETFLAGS} ../${DL_TIMER_DIR}/${DL_TIMER_NAME}" -C $@
+
 dl_timer:
 	make -C ${DL_TIMER_DIR} sm_lib
 
 clean:
 	${MAKE} -C tra_adv_serial $@
 	${MAKE} -C tra_adv_acc $@
 	${MAKE} -C tra_adv_omp $@
+	${MAKE} -C tra_adv_omp_gpu $@
 
 allclean:
 	rm -rf tra_adv_serial
 	rm -rf tra_adv_acc
+	rm -rf tra_adv_omp
+	rm -rf tra_adv_omp_gpu
```

Comment:
allclean didn't work from me because an old mod file in dl_timer, I think you need to add:
`${MAKE} -C ${DL_TIMER_DIR} allclean`

It not part of this PR but the problem is also present in the other folders of tracer_advection

Reply:
Done.